packages/api/lib/writeRecords/write-execution.js

```diff
@@ -90,18 +90,18 @@ const buildExecutionRecord = ({
 
 const writeExecutionToES = async (params) => {
   const {
-    dynamoRecord,
+    apiRecord,
     esClient = await Search.es(),
   } = params;
   try {
     await upsertExecution({
       esClient,
-      updates: dynamoRecord,
+      updates: apiRecord,
       index: process.env.ES_INDEX,
     });
-    logger.info(`Succesfully wrote Elasticsearch record for execution ${dynamoRecord.arn}`);
+    logger.info(`Succesfully wrote Elasticsearch record for execution ${apiRecord.arn}`);
```

Comment:
`logger.info(`Successfully wrote Elasticsearch record for execution ${apiRecord.arn}`);`